admin/acceptance.py

```diff
@@ -146,7 +147,7 @@ def start_nodes(self):
         }
         metadata.update(self.metadata)
 
-        for index in range(2):
+        for index in range(1):
```

Comment:
That's just to speed things up while I'm working on the DigitalOcean parts. I'll revert once the branch is ready.